lib/rolodex/route.ex

```diff
@@ -422,93 +422,51 @@ defmodule Rolodex.Route do
           verb: atom()
         }
 
-  @doc """
-  Checks to see if the given route matches any filter(s) stored in `Rolodex.Config`.
-  """
-  @spec matches_filter?(t(), any()) :: boolean()
-  def matches_filter?(route, filters)
-
-  def matches_filter?(route, filters) when is_list(filters) do
-    Enum.any?(filters, fn
-      filter_opts when is_map(filter_opts) ->
-        keys = Map.keys(filter_opts)
-        Map.take(route, keys) == filter_opts
-
-      filter_fun when is_function(filter_fun) ->
-        filter_fun.(route)
-
-      _ ->
-        false
-    end)
-  end
-
-  def matches_filter?(_, _), do: false
-
   @doc """
   Looks up a `Phoenix.Router.Route` controller action function, parses any
```

Comment:
Needs updating, I think.

Reply:
So it does, will do another quick audit of the lib docs